woocommerce/woocommerce
# Adjust copy and show the address in the updated warning message.

### Changes proposed in this Pull Request:


Closes #50556 

According to the design provided the following adjustments were made:

* Change the wording slightly to "Calculated at checkout", note that removing the text emphasis was already achieved in a [previous PR](https://github.com/woocommerce/woocommerce/pull/51608)
* Change the warning notice wording which includes putting the formatted address in the notice.


Before:

<img width="1409" alt="Screenshot 2024-10-25 at 4 43 15 PM" src="https://github.com/user-attachments/assets/5a028e78-6dfd-4405-9a13-ab1f493942e9">


After: 

<img width="1399" alt="Screenshot 2024-10-25 at 4 44 04 PM" src="https://github.com/user-attachments/assets/bb6b24a2-e197-4008-9423-7dae0bb72b29">




### How to test the changes in this Pull Request:

For the address warning:

1. Set up a shipping zone for any country
2. Add some products go to checkout, enter a full valid address that is not supported by the shipping zone.

For the updated wording:

1. Have the below settings:

<img width="974" alt="Screenshot 2024-10-25 at 4 48 32 PM" src="https://github.com/user-attachments/assets/ea4d05ee-3c72-4737-9673-ba8b59e1cb11">


2. As a user with no address entered yet, add some items to cart
3. Go to cart page, see the updated style and slight wording change (`Calculated at checkout`) 

Example:

<img width="1349" alt="Screenshot 2024-10-25 at 4 46 41 PM" src="https://github.com/user-attachments/assets/8af2f99f-9dc5-4a21-8c8b-fc87ae49be0e">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Adjust copy in the cart sidebar when the shipping calculator is not enabled.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
